### PR TITLE
contract.yml update

### PR DESCRIPTION
**Pull request by onrkskn**
I translate from English to Turkish. (Let me know if you find an error)

### CODE REVIEW

**Comment by microstudi** [comment]
Hi, first of all thanks.
Try to solve the errors, you can see that checks are failing.
If yo want to see the error, you can see the "Job Log" here:
https://travis-ci.org/GoteoFoundation/goteo/

In this case, it's a YAML format issue.
Also, you can avoid this kind of errors by using http://translate.goteo.org, commits then will be automatic (and you'll get added to the list of contributors in the README file). But if you prefer github directly, it's fine as well. Just make sure that tests are passing.

**Comment by onrkskn** [comment]
I compared Spanish and English. There are also line deficiencies in English. I wrote all the responses at line 38. I tried all the spelling forms, they did not work.

File: Resources/translations/tr/contract.yml [hunk 1]
```diff
@@ -1 +1,101 @@
...
+mandatory-contract-bank: 'Proje için bir banka hesabı belirtmek zorunludur'
+mandatory-contract-bank_owner: 'Banka hesabı sahibinin adını doldurması zorunludur'
+mandatory-contract-paypal_owner: 'Projenin PayPal hesabı sahibinin adını doldurması zorunludur'
+mandatory-contract-reg_date_2: 'Noterin kamu tüzüğünü onayladığı tarihi belirtmesi zorunludur.'
+mandatory-contract-reg_id_2: 'Noterin kayıt sayısını belirtmesi zorunludur'
+mandatory-contract-reg_idloc_2: 'Noterin çalışma yaptığı şehiri belirtmesi zorunludur'
+mandatory-contract-reg_idname_2: 'Kamusal şirket tüzüğüne izin veren noterin ismini belirtmek zorunludur. '
+mandatory-contract-reg_name_1: 'Kayıtların tam adının belirtilmesi zorunludur (örn., Madrid Ticaret Kaydı/Barselona Ticaret Kaydı/Sevilla Kooperatifi Kaydı)'
+mandatory-contract-reg_number_1: 'Kayıt numarasını belirtmek zorunludur'
+mandatory-contract-reg_number_2: 'Kayıt numarasını belirtmek zorunludur
```
Comment (microstudi) [inline Resources/translations/tr/contract.yml]: Here you have the problem, note that there's a missing `'` at the end

Comment (microstudi) [reply Resources/translations/tr/contract.yml]: And, if you look at the result of the Travis job, you will see the error:
```
There was 1 failure:
1) Goteo\Application\Tests\ConfigTest::testYamlLangFiles
YAML parse error in [/home/travis/build/GoteoFoundation/goteo/Resources/translations/tr/contract.yml]
Unexpected characters near "Proje sorumlu kişinin doğum tarihini girmek zorunludur'" at line 38 (near "mandatory-project-field-contract_birthdate: 'Proje sorumlu kişinin doğum tarihini girmek zorunludur'").
/home/travis/build/GoteoFoundation/goteo/tests/Goteo/Application/ConfigTest.php:37

```

File: Resources/translations/tr/contract.yml [hunk 2]
```diff
@@ -50,7 +50,7 @@ personal-field-post_address: 'Posta Adresi'
...
 tooltip-contract-bank_owner: 'Burada, banka hesabının sahibinin kim olduğunu belirtiniz.'
-tooltip-contract-docs: 'İhtiyacınız olan kadar çok dosya yükleyebilirsiniz, bunların her biri 2Mb'den büyük olmamalıdır. Tercihen bunları .PDF, .ODT, .DOC, .JPG formatında gönderiniz. Eğer bir tarayıcınız yoksa, her bir belgenin resmini çekebilirsiniz, ancak lütfen bilgilerin okunabilir olduğundan emin olunuz.'
+tooltip-contract-docs: 'İhtiyacınız olan kadar çok dosya yükleyebilirsiniz, bunların her biri 2Mb''den büyük olmamalıdır. Tercihen bunları .PDF, .ODT, .DOC, .JPG formatında gönderiniz. Eğer bir tarayıcınız yoksa, her bir belgenin resmini çekebilirsiniz, ancak lütfen bilgilerin okunabilir olduğundan emin olunuz.'
 tooltip-contract-entity_cif: 'Varlığın vergi numarasının hangisi olduğunu belirtiniz'
 tooltip-contract-entity_name: 'Sözleşmenin imzalanacağı taraflardaki varlığın tam adını belirtiniz.'
 tooltip-contract-entity_office: 'Varlık ile sizin pozisyonunuzun ne olduğunu belirtmeniz gerekmektedir.Onların tarafı adına hareket etme hakkının bulunduğunu belgeleyen belgeleri ('Dokümantasyon' bölümünde) eklemenizi isteriz.'
```
Comment (microstudi) [inline Resources/translations/tr/contract.yml]: Internal `'` simple quotes should be escaped with double simple quotes `''`

Comment (onrkskn) [reply Resources/translations/tr/contract.yml]: Thanks, I learned what travis is doing. I use it for the first time. I will pay more attention.

**Comment by microstudi** [comment]
Please, in the future create pull requests to the devel branch. 
Thanks

**Comment by onrkskn** [comment]
Did I throw it in the wrong place? I'll make it next time. Can you make merged request? I did not see

**Comment by microstudi** [comment]
The problem is that you cloned the branch l18_devel which is a branch created automatically by crowdin. I usually squash the commits on that branch, so history is lost. If I merge you there your contributions will be lost.
So please, clone the "devel" branch and do the pull requests there.
In these pull request I am merging manually to devel so don't worry on your work that you've already done.

**Pull request closed without merging**